.rubocop.yml

```diff
@@ -8,8 +8,8 @@ AllCops:
     - 'dashboard/scripts/**/*'
     - 'dashboard/db/**/*'
     - 'dashboard/db/*'
-  RunRailsCops: true
   DisplayCopNames: true
+  TargetRubyVersion: 2.2
```

Comment:
Thanks! It seems as if importing .rubocop_hound.yml caused significant changes to the file. Are we certain that none of the previous stuff is important?